mkdocs.yml

```diff
@@ -8,6 +8,12 @@ strict: true
 # https://www.mkdocs.org/user-guide/configuration/#remote_branch
 remote_branch: gh-pages
 
+# Hooks
+# https://www.mkdocs.org/user-guide/configuration/#hooks
+
+hooks:
+  - mkdocs-hooks/custom-edit-url.py
```

Comment:
is the file schema the same? ( expected function names)

Reply:
I don't understand what you mean by "file schema". I guess these parts of the `hooks:` documentation are relevant though:

## Plugin event handlers, without `self`

> Then the file `my_hooks.py` can contain any [plugin event handlers](https://www.mkdocs.org/dev-guide/plugins/#events) (without `self`), e.g.:
>
> ```python
> def on_page_markdown(markdown, **kwargs):
>     return markdown.replace('a', 'z')
> ```

## Replacing the `mkdocs-simple-hooks` plugin

The `hooks:` docs also mention replacing the `mkdocs-simple-hooks` plugin:

> You might have seen this feature in the [mkdocs-simple-hooks plugin](https://github.com/aklajnert/mkdocs-simple-hooks). If using standard method names, it can be directly replaced, e.g.:
> 
> ```diff
> -plugins:
> -  - mkdocs-simple-hooks:
> -      hooks:
> -        on_page_markdown: 'my_hooks:on_page_markdown'
> +hooks:
> +  - my_hooks.py
> ```

## Link to full docs for `hooks:`

https://www.mkdocs.org/user-guide/configuration/#hooks